books/centaur/sv/cosims/Makefile

```diff
@@ -94,12 +94,14 @@ TESTS := $(patsubst %/spec.sv,%,$(SPECFILES))
 TESTCERTS := $(patsubst %,%/ok,$(TESTS))
 OUTDATAS_NCV := $(patsubst %,%/outputs.ncv.data,$(TESTS))
 OUTDATAS_VCS := $(patsubst %,%/outputs.vcs.data,$(TESTS))
+OUTDATAS_IV  := $(patsubst %,%/outputs.iv.data,$(TESTS))
 
 PASSING_TESTS := $(filter-out $(FAILING_TEST_TARGETS), $(TESTCERTS))
 OUTDIRS_VCS := $(patsubst %,%/simv.daidir,$(TESTS))
 print_tests :
 	@echo $(TESTS)
 
+# %/ok : %/outputs.ncv.data %/outputs.vcs.data %/outputs.iv.data cosim-core
```

Comment:
@solswords / @jaredcdavis , you might consider making the inclusion of Icarus the default